src/components/phase/card/PhaseCard.tsx

```diff
@@ -0,0 +1,134 @@
+import classNames from 'classnames'
+import { useCallback, useEffect, useRef, useState } from 'react'
+import { useMediaType } from '../../../hook/useMediaType'
+import type { PhaseDTO, Duration, Task } from '../../../data/phase/dto.type'
+import ClearIcon from '@mui/icons-material/Clear'
+import LoupeIcon from '@mui/icons-material/Loupe'
+import moment from 'moment'
+import Link from 'next/link'
+
+type DropDownProps = Pick<PhaseDTO, 'phaseName' | 'tasks'> & {
+  refObj: React.RefObject<HTMLDivElement> | null
+  onClose: () => void
+  phaseDuration: Duration
+}
+
+export type PhaseCardProps = PhaseDTO
+
+export const DropDown: React.FC<DropDownProps> = ({
```

Comment:
I would have moved this component into a dedicated folder

Reply:
Yes you are right. It was something I was thinking of doing but decided against as the two components are strongly correlated. But having them apart helps fragment the code. It will be implemented